middlewares/entityValidation.js

```diff
@@ -1,20 +1,34 @@
 const { DOCUMENT_NOT_FOUND } = require('~/consts/errors')
 const { createError } = require('~/utils/errorsHelper')
 
-const isEntityValid = (entities) => {
+const isEntityValid = (entities, idSource = 'params') => {
   return async (req, res, next) => {
+    const models = []
+
     for (const { model, idName } of entities) {
```

Comment:
You can use reduce method of the array for this operation.

Reply:
do I have to tho? to me this way it looks more readable & understandable